x-pack/plugins/apm/server/routes/mobile/get_mobile_main_statistics_by_field.ts

```diff
@@ -40,10 +40,18 @@ export interface MobileMainStatisticsResponse {
     name: string | number;
     latency: number | null;
     throughput: number;
-    crashRate?: number;
+    crashRate: number;
   }>;
 }
 
+type MainStatistics = Array<{
```

Comment:
Also, I think this is not used

Reply:
This type is used in line 194, needed to set which type we want the array returned by merge()

Reply:
I see! I don't fully understand the following if you could please elaborate a bit more

```
export interface MobileMainStatisticsResponse {
  mainStatistics: Array<{
    name: string | number;
    latency: number | null;
    throughput: number;
    crashRate: number;
  }>;
}

type MainStatistics = Array<{
  name: string | number;
  latency: number | null;
  throughput: number;
  sessions: number;
  crashes?: number;
}>;
```

cause I would expect to be 
```
export interface MobileMainStatisticsResponse {
  mainStatistics: MainStatistics
}
```

why do you have to calculate the crashRate here ->https://github.com/elastic/kibana/blob/b09ec7a1197573441ea85a2da8673c10767a717f/x-pack/plugins/apm/server/routes/mobile/get_mobile_main_statistics_by_field.ts#L204

Reply:
The merge of the two query responses is different from the return we want, for instance, we need sessions to calculate the `crashRate`, but we don't need them to be returned by `getMobileMainStatisticsByField()`, the main statistics we need are name, latency, throughput, and crashRate.
We calculate the `crashRate` here, same as we also calculate the `throughput`

Reply:
Maybe the naming is confusing, I can give it a thought and find a more clear name for the type

Reply:
Oh I see the confusion, sessions shouldn’t be part of errors query, does it make sense now?